src/App.js

```diff
@@ -14,6 +15,7 @@ function App() {
         <Route path='/' element={<Login />} />
         <Route path='/allReviews' element={<AllReviews />} />
         <Route path='/allReviews/:review_id' element={<SingleReview />} />
+        <Route path='/allReviews/:review_id/comments' element={<AllComments />} />
```

Comment:
Do you really need a route to a seperate to a commnets page?
As you are loading the comments on the page for each review this route isn't really necessary

Reply:
I have changed this part, so I will merge now, thx